libs/platform/src/lib/form/multi-combobox/commons/base-multi-combobox.ts

```diff
@@ -368,6 +382,10 @@ export abstract class BaseMultiCombobox extends CollectionBaseInput implements A
         super(_cd, ngControl, ngForm, formField, formControl);
     }
 
+    ngOnInit(): void {
+        this._initializeDataSource(this._data);
```

Comment:
If datasource isn't set till this moment it wont be opened, and previously we had this in setted, why is this changed?

Reply:
So, initialization of the datasource, depends on the value of limitless property, which is passed using @Input decorator as well. Leaving DS initialization in the setter, will cause regression, for example: if users will call the component using the following order of the inputs ` <... [datasource]=".." [limitless]="..">`, we won't have limitless value ready for the initialization, that's why I moved it to the `ngOnInit`

Reply:
@dmitry-stepanenko I re-tested it and it's working without `ngOnChanges`

Reply:
@dmitry-stepanenko added button for toggling `limitless` input

Reply:
Yea, changing limitless will be working as we're processing then that change in `searchTermChanged` method. 

What i'm talking about - we're not reinitializing datasource on `dataSource` input change anymore. User has to set it once at the beginning and cannot change it. That's the regression.

Reply:
@platon-rov ah, got it, will handle this case

Reply:
@platon-rov done, please check again